### PR TITLE
Add developer information

### PR DESCRIPTION
**Pull request by mdedetrich**
@eed3si9n @mkurz @ktoso Let me know if this developer information is correct. I need to add something here otherwise I cannot publish to sonatype, see https://github.com/sbt/sbt-pull-request-validator/actions/runs/7681945912/job/20935773398#step:9:134

List is in alphabetical order

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -14,6 +14,33 @@ licenses := Seq("Apache-2.0" -> url("http://www.apache.org/licenses/LICENSE-2.0"
...
+    name = "Eugene Yokota",
+    email = "eed3si9n@gmail.com",
+    url = url("https://github.com/eed3si9n")
+  ),
+  Developer(
+    id = "eed3si9n",
+    name = "Eugene Yokota",
+    email = "eed3si9n@gmail.com",
+    url = url("https://github.com/eed3si9n")
+  ),
```
Comment (mkurz) [inline build.sbt:29]: You have Eugene twice in the list. Also not sure if he wants his e-mail published here (email can stay empty and publishing will still work).

Also I have seen various styles for this developers fields...
Alternative would be to put in something very generic like:
https://github.com/sbt/sbt-multi-jvm/blob/main/build.sbt#L45-L50

**Review by mkurz — APPROVED** [review]

**Comment by mdedetrich** [comment]
Ill just go with generic version, let me quickly fix

**Comment by mdedetrich** [comment]
@mkurz I have to head out now, will merge + deploy later

**Comment by mkurz** [comment]
I pushed the tag meanwhile, but GitHub is a bit slow currently: https://github.com/sbt/sbt-pull-request-validator/actions/runs/7682275000/job/20936422866

**Comment by mkurz** [comment]
@mdedetrich I am off for a couple of hours. When it's on maven central we need to publisht the release notes and we are done I guess.

**Comment by mdedetrich** [comment]
@mkurz Publishing now worked, also created the release notes as it should be on maven central now

**Pull request merged**